app/dashboard/src/providers/DriveProvider.tsx

```diff
@@ -0,0 +1,84 @@
+/** @file The React provider (and associated hooks) for providing reactive events. */
+import * as React from 'react'
+
+import invariant from 'tiny-invariant'
+import * as zustand from 'zustand'
+
+import { type DirectoryAsset } from 'enso-common/src/services/Backend'
```

Comment:
We have backend re-export in the project, shall we use the reexport or import directly from the `enso-common`?

Reply:
If we're not going to have wrappers around common backend, then let's use direct imports